src/eko/output/struct.py

```diff
@@ -91,7 +92,7 @@ def raw(self):
             elif isinstance(value, float):
                 value = float(value)
             elif isinstance(value, interpolation.XGrid):
-                value = value.dump()
+                value = value.dump()["grid"]
```

Comment:
Uhm, this is weird: the serialization of `XGrid` should be fully managed by the class itself.

So, we should always save what `.dump()` returns, and `XGrid` itself should be able to be loaded from the whole return value of a `.dump()`.
If this is not happening, we should fix on `XGrid` side, not here.

Reply:
Yes indeed, the point is that `dump()` return a dictionary like {'grid':[...], 'log':[...]} where the actual grid is inside `grid`. For the tests, sometimes the `log` part is used and sometimes it is not (and this is causing errors). So I was not sure on how to fix

Reply:
We should make sure that the `log` part is always used, or ignored, on the consumer side